gridscale/provider.go

```diff
@@ -22,10 +22,16 @@ func Provider() terraform.ResourceProvider {
 			},
 			"api_url": {
 				Type:        schema.TypeString,
-				Required:    true,
-				DefaultFunc: schema.EnvDefaultFunc("GRIDSCALE_URL", "https://api.gridscale.io"),
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("GRIDSCALE_URL", nil),
 				Description: "the url for the gridscale API.",
 			},
+			"gsc_timeout_secs": {
```

Comment:
I would simply name it `timeout` and in docs. we should the time unit in seconds